styles.css

```diff
@@ -188,5 +209,5 @@ footer > a {
 }
 
 footer > a:hover {
-  color: (245, 252, 252);
+  color: white;
```

Comment:
Maybe you could add for the icons that they take an opacity value of 0.5 when hovering.

Reply:
very nice idea, i will try to make it